pages/install-device-page.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class InstallDevicePage {
+    readonly page: Page;
+    readonly secondDeviceItem: Locator;
+    readonly acceptButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.secondDeviceItem = page.getByLabel('Available devices').locator('svg').nth(2);
+        this.acceptButton = page.getByRole('button', { name: 'Accept' });
+    }
+
+    async changeInstallDevice() {    
+        await this.secondDeviceItem.click();
+    }
+
+    async acceptInstallDeviceChange() {
```

Comment:
you don't need to repeat "InstallationDevice" in the method name, when you call it is clear who is the caller, accept() is enough.

Reply:
I'm trying to be consistent in naming with what I see in other tests, but each test is different.